webapp/channels/src/components/signup/signup.tsx

```diff
@@ -604,6 +613,12 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
                     onDismiss: dismissAlert,
                 });
                 setIsWaiting(false);
+
+                // Special case for accessibility to show the error message when the username is already taken
```

Comment:
Could we perhaps remove the special case if we gave that banner `role="alert"`?

It looks like that's all you need to make an [alert banner](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) according to the APG

Reply:
I think this falls in line with the rest of the behaviour in which you still want to focus the input, so I think we should leave this one there.